lsp/pom.xml

```diff
@@ -19,7 +19,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>21.0</version>
```

Comment:
The current version is `29.0-jre`. You don't want to update it to the latest?

Reply:
Any update > 24.1.1 would resolve the security vulnerability. I wasn't sure if we wanted to jump major versions all that much without causing issues, but I am happy to update to the latest and test it.

Reply:
Got it. This looks good.